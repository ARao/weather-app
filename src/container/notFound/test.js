import React from 'react';
import { shallow } from 'enzyme';
import { NotFound } from './index';


describe('Not Found', () => {
  const errorCode = 400;
  let wrapper
  beforeEach(() => {
    wrapper = shallow(<NotFound status="404" statusText="Not Found" />);
  })

  it('++++ render NotFound', () => {
    expect(wrapper.length).toEqual(1);
  });
  it('++++ snapshot match', () => {
    expect(wrapper).toMatchSnapshot();
  });
});