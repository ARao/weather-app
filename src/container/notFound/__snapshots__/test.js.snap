// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found ++++ snapshot match 1`] = `
<ErrorView
  status="404"
  statusText="Not Found"
/>
`;
